evap/evaluation/migrations/0065_questionnaire_type.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.0.2 on 2018-03-12 18:31
+
+from django.db import migrations, models
+
+
+def migrate_is_for_contributors_to_type(apps, schema_editor):
+    Questionnaire = apps.get_model('evaluation', 'Questionnaire')
+    Questionnaire.objects.filter(is_for_contributors=True).update(type=1)
+
+
+def migrate_type_to_is_for_contributors(apps, schema_editor):
+    Questionnaire = apps.get_model('evaluation', 'Questionnaire')
+    Questionnaire.objects.filter(type=1).update(is_for_contributors=True)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0064_questionnaire_index_rename'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='questionnaire',
+            options={'ordering': ('type', 'order', 'name_de'), 'verbose_name': 'questionnaire', 'verbose_name_plural': 'questionnaires'},
+        ),
+        migrations.AddField(
+            model_name='questionnaire',
+            name='type',
+            field=models.IntegerField(choices=[(0, 'Top questionnaire'), (1, 'Contributor questionnaire'), (2, 'Bottom questionnaire')], default=0, verbose_name='type'),
+            preserve_default=True,
```

Comment:
we haven't discussed this yet, but I think that is the dropout questionnaire by @jakobmache and the single result questionnaire are probably going to have their own type. the evaluation feedback (#212) might also get a new questionnaire type.

Now the question is, if we add those types, do we simply append those to the list of questionnaire types, since the ordering for them is irrelevant since they are special-cased anyways? Or do we want them in a certain position in the list (e.g. dropout questionnaire at the very top since it will be shown at the very top)? In that case I would leave generous gaps between the numbers to not require migrations for rewriting all the types on each change. But I think I'm fine with appending. What do you think?

Reply:
I guess @janno42 will have to decide on the ordering that will be required. Having the dropout questionnaire at the top seems logical to me. Which values would you set right now if you had to leave gaps? 1000, 2000 and 3000?

Reply:
well, the ordering in the UI is technically irrelevant, since those special questionnaires are most likely special cased and explicitly inserted at some specific place. it's more a semantic thing. if we want the type to match the ordering even for those types where the type is not actually used for ordering.

initially i thought of 10, 20 and 30. but in the end it does not matter at all, you choose :)

Reply:
10, 20, 30 it is :)